src/main/java/CycleTimeGUI.java

```diff
@@ -29,45 +40,89 @@ public CycleTimeGUI(int projectID,String authToken, String TAIGA_API_ENDPOINT,St
         this.firstDate = firstDate;
         this.lastDate = lastDate;
         System.out.println(projectID+"\n"+authToken+"\n"+TAIGA_API_ENDPOINT+"\n"+firstDate+"\n"+lastDate);
-//        return;
+        this.xAxis = new CategoryAxis();
+        this.yAxis = new NumberAxis(0, 12,1);
+        this.scatterChart = new ScatterChart<>(xAxis, new NumberAxis());
     }
 
 
 
     public static void main(String[] args) {
-
 //        launch(args);
     }
 
     @Override
     public void start(Stage primaryStage) {
-        Stage window=new Stage();
-        window.initModality(Modality.APPLICATION_MODAL);
-        window.setTitle("Cycle Time Chart");
-        window.setMinWidth(300);
-
-        TextField sprintInput = new TextField("Enter Sprint");
-        sprintInput.setPrefWidth(100);
-        sprintInput.setMaxWidth(100);
-
-        Button submitButton=new Button("Submit");
-
-        submitButton.setOnAction(e->{
-            String sprint = sprintInput.getText();
-            String TAIGA_API_ENDPOINT = "https://api.taiga.io/api/v1";
-            List<JsonNode> sprintDetails = Burndown.getMilestoneStats(authToken, TAIGA_API_ENDPOINT, projectID, sprint);
```

Comment:
why are we calling burnDown?

Reply:
if it is commonly required, better add the method to some common util package

Reply:
Calling Burndown to get the sprint start date and end date

Reply:
Not sure about the usage in the future.
We will refactor if needed

Reply:
ok then, will mark it under code debt